content/python.rst

```diff
@@ -140,18 +140,23 @@ NO
   Unicode is a character set and UTF-8 is a particular way of
   encoding Unicode.
 
-* When a method does not use the *self* attribute, this is a code smell
-  that this method should be placed somewhere else.
+* When a method does not use the *self* attribute, this is a sure sign
+  that this method should be placed somewhere else. Also, it's a code smell.
 
 
-Abstract
-========
+Abstract classes - Twisted style
+================================
+
+The descriptions used below for abstract classes follows the 'definition' 
+used by the Twisted framework. While not exactly in line with cannonical Python
+definitions (that mention metaclasses and the use of decorators) these points
```

Comment:
I agree that maybe this paragraph is a bit pedantic. The point I was trying to convey was that, if you are someone coming from a more classical Python background, you should look at these classes like "textbook" abstract classes and not strictly Pythonic abstract classes. 

Metaclasses in Python have a very different functionality as compared with Twisted's Abstract classes. For example, when defining a metaclass, if there's any method decorated as an abstractmethod, any implementation of that class needs to override the abstract method in order to be allowed to construct an instance. It basically delegates worries about overriding to the interpreter, not the developer.

Twisted (and Zope in the background) chose a different road, where the developer has to handle his own implementations of abstract classes. Because of this, you have a different catching point for any exceptions that might slip through.